content/learn/protocol-parameters.md

```diff
@@ -311,3 +282,39 @@ The character limit of transaction memos.
 **Current Value:** 8
 
 The maximum number of signatures that a multi-sig account can have.
+
+
+## Governance Module
+
+These parameters control [governance](/community/governance/) on the Pocket Network.
+
+### ACL
+
+Access control list for updating the on-chain parameters. Currently all parameters are owned and managed by the [`DAOOwner`](#daoowner) address.
+
+### DAOOwner
+
+**Current Value:** a83172b67b5ffbfcb8acb95acc0fd0466a9d4bc4
+
+The account which has the permission to submit governance transactions on behalf of the DAO.
+
+### Upgrade
+
+A list consisting of the block height at which the most recent version of Pocket Core was activated, the block height of the previous version, and the features activated in this version.
```

Comment:
Suggestion:

**Current Value:**
- Height: 68849
- Version: 0.9.0
- OldUpgradeHeight: 57616
- Features:
  - MREL: 69232
  - NCUST: 0
  - REDUP: 57620
  - RSCAL :69232


An object describing the details of a protocol upgrade, consisting of the following fields:

- Height: Specifies when the upgrade was applied
- Version: The version that the upgrade brought
- OldUpgradeHeight: When the previous upgrade was last applied.
- Features: A list of feature flags, and the heights at which they should be activated at

Reply:
@blockjoe Great idea. I can totally add that, but I'd want to define what `MREL`, `NCUST` etc. are too. Got a link or a definition list?

Reply:
@protocol team for this, this is just the live value that I need to understand.